docs/data/date-pickers/date-picker/date-picker.md

```diff
@@ -11,6 +11,16 @@ materialDesign: https://material.io/components/date-pickers
 
 <p class="description">The date picker let the user select a date.</p>
 
+:::success
+This component has received several major improvements on the new **v6 beta**:
+
+- [Fields: the new default input for pickers](/x/react-date-pickers/fields/).
```

Comment:
We should probably have absolute links in these cases?
Relative ones will link to "nowhere" for v5 docs 🙈

Reply:
Done